style/outsidein.css

```diff
@@ -164,7 +173,8 @@ body {
 }
```

Comment:
I'm not sure that upper-roman is working very well with this table of contents, @jhung, at least not as configured here. Perhaps just bullet points?